applications/mne_fiff_anonymizer/fiffanonymizerapp.cpp

```diff
@@ -0,0 +1,631 @@
+
+
+//*************************************************************************************************************
+//=============================================================================================================
+// Eigen INCLUDES
+//=============================================================================================================
+
+
+//*************************************************************************************************************
+//=============================================================================================================
+// USED NAMESPACES
+//=============================================================================================================
+using namespace MNEFIFFANONYMIZER;
+
+//*************************************************************************************************************
+//========================================================= ====================================================
+// DEFINE GLOBAL METHODS
+//=============================================================================================================
+
+
+//*************************************************************************************************************
+//=============================================================================================================
+// DEFINE MEMBER METHODS
+//=============================================================================================================
+
+FiffAnonymizerApp::FiffAnonymizerApp()
+    : version(1.0),
+      maxValidFiffVerion(1.3),
+      versionStr(QString::number(version)),
+      name("MNE Fiff Anonymizer"),
+      description("Application that removes or modifies Personal Health Information or Personal Identifiable information."),
+      m_bVerboseMode(false),
+      m_bBruteMode(false),
+      m_bQuietMode(false),
+      m_bDeleteInputFileAfter(false),
+      m_bDeleteInputFileConfirmation(true),
+      m_sDfltString("mne_fiff_anonymizer"),
+      m_dateDfltDate(QDateTime(QDate(2000,1,1), QTime(1, 1, 0))),
+      m_iMeasurementDayOffset(0),
+      m_bUseMeasurementDayOffset(false),
+      m_iSubjectBirthdayOffset(0),
+      m_bUseSubjectBirthdayOffset(false),
+      m_dateMeasurmentDate(m_dateDfltDate),
+      m_date_IdDate(m_dateDfltDate),
+      m_iDfltSubjectId(0),
+      m_sDfltSubjectFirstName(m_sDfltString),
+      m_sDfltSubjectMidName("x"),
+      m_sDfltSubjectLastName(m_sDfltString),
+      m_dateSubjectBirthDay(m_dateDfltDate),
+      m_iDfltSubjectWeight(0),
+      m_iDfltSubjectHeight(0),
+      m_sDfltSubjectComment(m_sDfltString),
+      m_sDfltSubjectHisId(m_sDfltString),
+      m_iDfltProjectId(0),
+      m_sDfltProjectName(m_sDfltString),
+      m_sDfltProjectAim(m_sDfltString),
+      m_sDfltProjectPersons(m_sDfltString),
+      m_sDfltProjectComment(m_sDfltString),
+      m_sFileNameIn(""),
+      m_sFfileNameOut(""),
+      m_printInSameLineHelper(nullptr),
```

Comment:
`nullptr `was introduced in C++11, which is not supported by the minimalVersion. Use `Q_NULLPTR` instead to init raw pointers. If you decide to use a guarded pointer (see comment below) this line of code is obsolete.

Reply:
it is no longer a pointer, so no it is initialized with a qDebug() call to generate a QDebug obj.
Thanks.